deno.jsonc

```diff
@@ -1,3 +1,6 @@
 // XXX
 // This file is required to workaround the following issue
 // https://github.com/vim-denops/denops.vim/issues/218
+{
+  "lock": false
+}
```

Comment:
Why do we need this?

Reply:
When I was started denops server in the root directory, `deno.lock` file is generated because of `deno.jsonc`.
Yes, this is workaround, but I don't know where denops server is running.